### PR TITLE
Add missing failover params

### PR DESCRIPTION
**Pull request by lowitea**
Add the missing parameters to the failover section:

```yml
  vars:
    cartridge_failover_params:
      failover_timeout: <<sec>>
      fencing_enabled: <<bool>>
      fencing_timeout: <<sec>>
      fencing_pause: <<sec>>
```

### CODE REVIEW

File: README.ru.md [hunk 1]
```diff
@@ -290,9 +290,13 @@ hosts:
 
 # параметры фейловера
 failover:
-  mode: stateful                      # (опционально) вариант работы фейловера (stateful, eventual, disabled)
-  state_provider: stateboard          # (опционально) провайдер предоставляющий фейловер (stateboard, stateful)
-  stateboard_params:                  # (опционально) параметры для провайдера
+  mode: stateful              # (опционально) вариант работы фейловера (stateful, eventual, disabled)
+  state_provider: stateboard  # (опционально) провайдер предоставляющий фейловер (stateboard, stateful)
+  failover_timeout: 60        # (опционально) время (в секундах), определяет время перевода инстанса в статус `dead`
+  fencing_enabled: true       # (опционально) включение режима fencing
```
Comment (ftelnov) [inline README.ru.md:296]: There is much less information available comparing to the English doc you wrote. Let's close the gap, especially cuz we prioritize Russian docs a little more.

Comment (lowitea) [reply README.ru.md]: You're right, it's done

File: README.ru.md [hunk 2]
```diff
@@ -290,9 +290,14 @@ hosts:
 
 # параметры фейловера
 failover:
-  mode: stateful                      # (опционально) вариант работы фейловера (stateful, eventual, disabled)
-  state_provider: stateboard          # (опционально) провайдер предоставляющий фейловер (stateboard, stateful)
-  stateboard_params:                  # (опционально) параметры для провайдера
+  mode: stateful              # (опционально) вариант работы фейловера (stateful, eventual, disabled)
+  state_provider: stateboard  # (опционально) провайдер предоставляющий фейловер (stateboard, stateful)
+  failover_timeout: 60        # (опционально) время (в секундах), определяет время перевода инстанса в статус `dead`
```
Comment (ftelnov) [inline README.ru.md:295]:
```suggestion
  failover_timeout: 60        # (опционально) время (в секундах), определяет время перевода инстанса со статусом `suspect` в статус `dead`
```

**Review by ftelnov — CHANGES_REQUESTED** [review]
Generally LGTM, review suggestions to the RU docs and we are good to go

**Review by ftelnov — APPROVED** [review]
LGTM

**Pull request merged**